src/json.cpp

```diff
@@ -6,7 +6,7 @@
 #include "rapidjson/prettywriter.h"
 #include "rapidjson/error/en.h"
 
-const Uint32 BinaryFormatTag = 'spff';
+const Uint32 BinaryFormatTag = *"spff";
```

Comment:
This was actually technically legal (although StackOverflow warns the reader about portability)

Reply:
Legal or not, my objective was to get GCC to shut up when compiling a release build. Aside from the explosion from the sound engine not being done getting replaced (comment lines 2828 to 2851 in interface/consolecommand.cpp) it now compiles completely silently.

Debug builds are a different problem. I may or may not get around to cleaning that up, but there's a lot of work to do, there.

Reply:
Understandable. I'm sad to admit the entire codebase is a royal mess, I'd rather not even think about debug builds right now.